packages/data-broker/src/services/dhis/pullers/EventsPuller.js

```diff
@@ -28,17 +29,56 @@ export class EventsPuller {
       endDate,
       dataElementIdScheme: 'code',
     });
-
-    const translatedEventAnalytics = await this.translator.translateInboundEventAnalytics(
-      eventAnalytics,
+    const orgUnitEventAnalytics = await this.translator.translateInboundEventAnalytics(
+      orgUnitResults,
       dataElementSources,
     );
 
-    return buildEventsFromDhisEventAnalytics(
-      this.models,
-      translatedEventAnalytics,
-      dataElementCodes,
-    );
+    return buildEventsFromDhisEventAnalytics(this.models, orgUnitEventAnalytics, dataElementCodes);
+  };
+
+  pullEventsForApi = async (api, programCode, options) => {
```

Comment:
Not related to your PR but could we add `@private` tags to these methods, will help Kostas' PR when adding types

Reply:
Good idea, will add 👍